model/host/db.go

```diff
@@ -239,19 +239,23 @@ func runningHostsQuery(distroID string) bson.M {
 func hostsCanRunTasksQuery(distroID string) bson.M {
 	distroIDKey := bsonutil.GetDottedKeyName(DistroKey, distro.IdKey)
 	bootstrapKey := bsonutil.GetDottedKeyName(DistroKey, distro.BootstrapSettingsKey, distro.BootstrapSettingsMethodKey)
+
 	return bson.M{
-		distroIDKey:  distroID,
-		StartedByKey: evergreen.User,
 		"$or": []bson.M{
 			{
-				StatusKey: evergreen.HostRunning,
+				distroIDKey:  distroID,
```

Comment:
I'd add a comment here to explain why you did this, so a well-intentioned engineer doesn't come by and try to simplify things

Reply:
Done!